src/backend/images_cache.ts

```diff
@@ -34,8 +34,11 @@ const getImageFromCache = (url: string) => {
     pending.set(
       digest,
       new Promise((res) => {
-        axios
-          .get(realUrl, { responseType: 'stream' })
+        axios({
+          method: 'get',
+          url: realUrl,
+          responseType: 'stream'
+        })
```

Comment:
Huh why was *this* necessary? Shouldn't `axios.get` and `axios` with `method: 'get'` do the same thing?

Reply:
It is the same thing. I was trying different things for an exception that was thrown here. I forgot to revert the change. I'll do just that